src/io/flutter/perf/FlutterWidgetPerfManager.java

```diff
@@ -186,7 +186,10 @@ private boolean isProfilingEnabled() {
   private void debugActive(Project project, FlutterViewMessages.FlutterDebugEvent event) {
     debugIsActive = true;
 
-    assert (app != null);
```

Comment:
how is debug active without an app? It would be nice to get to the root cause.

Reply:
I'm not sure, but from the stack trace, it seemed that the line after the assert (`app.addStateListener(this);`) was triggering the NPE.